src/main/charts/confluence/templates/_helpers.tpl

```diff
@@ -110,6 +110,13 @@ Pod labels
 -Dconfluence.cluster.hazelcast.listenPort={{ .Values.confluence.ports.hazelcast }}
 {{- end }}
 
+{{- define "confluence.sysprop.s3Config" -}}
+{{- if and .Values.confluence.s3AttachmentsStorage.bucketName .Values.confluence.s3AttachmentsStorage.awsRegion }}
+-Dconfluence.filestore.attachments.s3.bucket.name={{ .Values.confluence.s3AttachmentsStorage.bucketName }}
+-Dconfluence.filestore.attachments.s3.bucket.region={{ .Values.confluence.s3AttachmentsStorage.awsRegion }}
```

Comment:
We should probably also include the ability to use `endpointOverride` as in Kubernetes environment I can imagine using minio will be a nice "experimental" feature.

Reply:
Added